media-sound/jack2/jack2-1.9.11_rc1.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="threads(+)"
+inherit eutils python-single-r1 waf-utils multilib-minimal
+
+DESCRIPTION="Jackdmp jack implemention for multi-processor machine"
+HOMEPAGE="http://jackaudio.org/"
+
+RESTRICT="mirror"
```

Comment:
Since it's pulling from GitHub directly that would make sense, right? The file isn't available from the gentoo mirrors. Or does that happen automatically somehow?